packages/protocol/schema.graphql

```diff
@@ -443,6 +443,11 @@ type SavingsContract @entity {
   are ERC20-compatible tokens.
   """
   token: Token
+
+  """
+  TODO: caption?
```

Comment:
Yes, please add this

Percentage of assets that should be held in the connector, determined by governance